### PR TITLE
Updated config file for Run 3 Kfold training

### PR DESCRIPTION
**Pull request by nikitapond**
Modifies the default config to create final kfold training sample for run 3 trainings.

### CODE REVIEW

**Comment by nikitapond** [comment]
Confirmed that this config runs fine, we end up with 260m jets (same as previous set of kfold trainings) so no loss of stats.

**Comment by samvanstroud** [comment]
> Confirmed that this config runs fine, we end up with 260m jets (same as previous set of kfold trainings) so no loss of stats.

Great thanks! When the linting is fixed I'm happy to merge

File: upp/configs/single-b.yaml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 global_cuts:
   train:
-    - [eventNumber, "%10<=", 7]
+    - [jetFoldHash, "%4!=", 0]
+    - [eventNumber, "%10<=", 8]
   val:
-    - [eventNumber, "%10==", 8]
-  test:
+    - [jetFoldHash, "%4!=", 0]
     - [eventNumber, "%10==", 9]
```
Comment (samvanstroud) [inline upp/configs/single-b.yaml:25]: If we wanted to make the training stats a little bit higher I suppose we could have split the holdout set into test and val, but this is fine too

**Comment by samvanstroud** [comment]
I think maybe https://github.com/umami-hep/puma/pull/204#issuecomment-1822610369 was posted in the wrong place.

Maybe you can just add the different folds as comments?

**Comment by nikitapond** [comment]
> I think maybe [umami-hep/puma#204 (comment)](https://github.com/umami-hep/puma/pull/204#issuecomment-1822610369) was posted in the wrong place.
> 
> Maybe you can just add the different folds as comments?

Whoops my bad, too many tabs open! Yeah good idea I'll add them as comments

File: upp/configs/single-b.yaml (hunk 1 above)
Comment (nikitapond) [reply upp/configs/single-b.yaml:31]: I also don't think the ~10% ish extra stats we get will have a significant effect on the models performance.

Comment (samvanstroud) [reply upp/configs/single-b.yaml:31]: good points!

**Review by samvanstroud — APPROVED** [review]

**Comment by samvanstroud** [comment]
Thanks @nikitapond ! Feel free to merge when you are ready

**Comment by nikitapond** [comment]
> Thanks @nikitapond ! Feel free to merge when you are ready

I don't think I have write access, so doesn't look like I can merge.

**Comment by samvanstroud** [comment]
> > Thanks @nikitapond ! Feel free to merge when you are ready
> 
> I don't think I have write access, so doesn't look like I can merge.

Should be fixed now :)

**Pull request merged**